### ISSUE TITLE
How to continue job with different learning rate?

### ISSUE THREAD

**Issue by zungam** [matterport/Mask_RCNN#681]
Hi, when you change the learning rate on a job, the loss function suddenly goes into the roof.
It seems like all weights are set to init again, or something, and you have to start over. Why? I would like to keep my weights, my model, my low loss, but just change the learning rate.

Best regards
zungam

**Comment by bahmed11** [comment]
I am using RunOnPlateau to reduce the learning rate if validation loss didn't improve for 5 subsequent epochs. You can set the patience to more if you would like.
I believe when you load the last trained model everything is starting from the same point
